### ISSUE TITLE
Removal of special comments in border

### ISSUE THREAD

**Issue by jorenvanhee** [ben-eb/postcss-discard-comments#18]
My special comments, if used with border, are getting removed.

Original:
```css
.buggy {
  color: #999 /*!theme-color*/;
  border: 1px solid #999 /*!theme-color*/;
  background-color: #999 /*!theme-color*/;
}
```

Minified:
```css
.buggy{color:#999 /*!theme-color*/;border:1px solid #999;background-color:#999 /*!theme-color*/}
```

**Comment by ben-eb** [comment]
Looks like a bug with postcss-ordered-values. You can workaround this for now by passing `{orderedValues: false}` as an options object, or `--no-orderedValues` from the command line.

**Comment by jorenvanhee** [comment]
Thanks! Workaround does its job.

**Comment by ben-eb** [comment]
@jorenvanhee Fixed by postcss-ordered-values `2.1.1`. Thanks for the report. 👍

**Comment by jorenvanhee** [comment]
👍  Thanks!
